packages/web-pkg/src/components/ContextActions/ActionMenuItem.vue

```diff
@@ -4,10 +4,11 @@
       v-oc-tooltip="showTooltip || action.hideLabel ? action.label(filterParams) : ''"
       :type="action.componentType"
       v-bind="componentProps"
-      :class="[action.class, 'action-menu-item']"
+      :class="[action.class, 'action-menu-item', 'oc-py-s', 'oc-px-m', 'oc-width-1-1']"
```

Comment:
My personal feeling is that `oc-px-s` (or just `oc-p-s` for setting all paddings to padding size `s`) is easier for the eye. What was your idea to use `m`? 🤔

Reply:
Absolutely love the UX effect of `oc-width-1-1` here. Well done!

Reply:
This is the padding setting for the action items in context menu. I tried to standardize the look and feel for action lists.

Reply:
Yeah ok, then let's keep this! thx